lexisnexis-api-client.gemspec

```diff
@@ -29,9 +28,10 @@ Gem::Specification.new do |s|
   s.add_dependency('dotenv')
   s.add_dependency('typhoeus')
 
+  s.add_development_dependency('activesupport')
```

Comment:
I am looking real hard for where we use activesupport but can't seem to find it. (Not that it matters; Rails is gonna pull it in in the app anyway).

Reply:
I'm using `present?` through a `be_present` matcher in `threat_metrix/verification_request_spec.rb`

Reply:
And it's just a development dependency for specs, like you said, doesn't matter for the app anyways